src/components/Configure/content/fields/FieldDefaultValueMapping/FieldDefaultValueTable.tsx

```diff
@@ -18,9 +21,25 @@ type FieldDefaultValue = {
   defaultValue: string,
 };
 
+/**
+ *
+ * gets field displayName from hydrated revision, object.allFields
+ * fallback to field name if not found
+ *  */
+const getDisplayNameFromField = (
+  field: string,
+  objectName: string,
+  readAction: HydratedIntegrationRead,
+) => {
+  const object = readAction && getObjectFromAction(readAction, objectName);
+  const allFields = object?.allFields as HydratedIntegrationFieldExistent[] || [];
```

Comment:
we now have `allFieldsMetadata` which is a map already, not a list so  you should use that and then you don't need to call `find` in the next line.